package.json

```diff
@@ -4,7 +4,7 @@
   "private": false,
   "license": "MIT",
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 3007",
```

Comment:
Instead of adding this here you can run it like `npm run dev -- -p 3007` or `yarn dev -p 3007`. Please remove.